cfgov/unprocessed/apps/filing-instruction-guide/css/main.less

```diff
@@ -209,6 +209,30 @@
   } );
 }
 
-#ctrl-f-modal h3 {
-  font-size: unit(16px / @base-font-size-px, em);
+#ctrl-f-modal {
```

Comment:
Should this go into the ctrl-f package itself?

Reply:
I opted to not put any CFPB styles into the ctrl-f package and instead apply them here. I think in a future release I'll add a config option to ctrl-f that lets the developer enable/disable the styles (`useTheme: cfpb` or `enableStyles: true` or something) so that these CSS rules can live inside the package but be optionally disabled. For now it's a little messy while we iterate on the design of the search tool.